packages/api/tsconfig.tsbuildinfo

```diff
@@ -1 +1 @@
46,3],[347,3],[98,3],[256,3],[97,3],[352,135],[353,135],[355,136],[354,137],[356,135],[358,138],[360,135],[357,135],[359,138],[361,3],[362,139],[348,3],[351,140],[363,46],[364,87],[365,3],[366,3],[350,3],[367,46],[368,141],[101,142],[369,50],[370,3],[371,143],[372,144],[374,145],[373,3],[375,3],[376,46],[377,146],[378,3],[379,147],[267,3],[106,148],[105,3],[295,149],[296,150],[349,3],[107,3],[95,46],[248,3],[249,151],[250,152],[247,153],[140,154],[125,50],[126,51],[127,51],[128,51],[129,51],[130,51],[131,51],[132,51],[133,51],[134,51],[135,51],[136,51],[137,51],[138,51],[139,51],[117,155],[116,3],[115,3],[114,3],[113,3],[96,3],[108,156],[142,157],[111,158],[109,159],[110,3],[112,160],[9,3],[10,3],[14,3],[13,3],[3,3],[15,3],[16,3],[17,3],[18,3],[19,3],[20,3],[21,3],[22,3],[4,3],[5,3],[26,3],[23,3],[24,3],[25,3],[27,3],[28,3],[29,3],[6,3],[30,3],[31,3],[32,3],[33,3],[7,3],[34,3],[35,3],[36,3],[37,3],[8,3],[42,3],[38,3],[39,3],[40,3],[41,3],[2,3],[1,3],[43,3],[12,3],[11,3],[252,161],[253,162],[251,3],[258,163],[260,164],[259,165],[261,166],[257,167],[254,168],[121,169],[119,170],[174,171],[118,3],[120,172],[291,173],[270,174],[151,175],[271,176],[268,177],[149,178],[269,179],[150,180],[279,181],[152,182],[280,183],[264,184],[148,185],[265,186],[146,187],[155,188],[172,189],[289,190],[292,191],[290,192],[282,193],[171,194],[283,195],[281,196],[156,157],[170,197],[122,198],[266,199],[262,189],[147,200],[298,201],[285,202],[145,203],[286,204],[284,205],[144,206],[287,207],[153,208],[288,209],[263,210],[143,211],[299,212]],"semanticDiagnosticsPerFile":[173,302,300,181,180,179,184,183,189,190,224,225,226,227,176,177,178,175,203,204,205,207,206,191,228,232,230,229,231,196,198,201,199,200,233,192,186,185,182,213,209,214,215,216,208,223,218,217,219,210,220,221,212,222,211,240,238,239,241,236,235,234,197,237,242,243,245,244,187,246,202,188,195,193,194,305,301,303,304,306,307,103,308,293,309,310,102,123,294,124,318,316,317,311,315,312,314,99,104,154,319,320,322,323,324,328,329,325,326,327,330,331,332,333,334,335,336,313,275,272,273,274,278,276,277,337,338,255,339,158,159,157,160,161,162,163,164,165,166,167,168,169,100,321,297,141,341,342,340,44,45,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,46,93,62,63,64,94,65,66,67,68,69,70,71,72,73,74,75,76,78,77,79,80,81,82,83,84,85,86,87,88,89,90,91,92,343,344,345,346,347,98,256,97,352,353,355,354,356,358,360,357,359,361,362,348,351,363,364,365,366,350,367,368,101,369,370,371,372,374,373,375,376,377,378,379,267,106,105,295,296,349,107,95,248,249,250,247,140,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,117,116,115,114,113,96,108,142,111,109,110,112,9,10,14,13,3,15,16,17,18,19,20,21,22,4,5,26,23,24,25,27,28,29,6,30,31,32,33,7,34,35,36,37,8,42,38,39,40,41,2,1,43,12,11,252,253,251,258,260,259,261,257,254,121,119,174,118,120,291,270,151,271,268,149,269,150,279,152,280,264,148,265,146,155,172,289,292,290,282,171,283,281,156,170,122,266,262,147,298,285,145,286,284,144,287,153,288,263,143,299]},"version":"4.6.2"}
```

Comment:
Should we really check in this file? Running `yarn workspace api build` produced no changes here locally until I deleted tsbuildinfo.

Reply:
Good catch -- I think it wasn't being ignored in my vscode git client until I added another line break. Fixed.